Makefile

```diff
@@ -149,63 +156,55 @@ kind-load-image:
 kind-e2e-test: multi-up e2e-test
 
 single-up: cleanup build manifests kustomize docker-build create-kind-cluster kind-load-image cert-manager
-ifeq ($(CLUSTER_SCOPE),true)
-	$(KUSTOMIZE) build config/deployments/control-plane-cluster-scope | kubectl apply --server-side --force-conflicts -f -
-else
-	$(KUSTOMIZE) build config/deployments/control-plane | kubectl apply --server-side --force-conflicts -f -
-endif
+	$(KUSTOMIZE) build config/deployments/control-plane$(DEPLOY_TARGET) | kubectl apply --server-side --force-conflicts -f -
 
 single-reload: build manifests kustomize docker-build kind-load-image cert-manager
 	kubectl config use-context kind-k8ssandra-0
-ifeq ($(CLUSTER_SCOPE),true)
-	$(KUSTOMIZE) build config/deployments/control-plane-cluster-scope | kubectl apply --server-side --force-conflicts -f -
-else
-	$(KUSTOMIZE) build config/deployments/control-plane | kubectl apply --server-side --force-conflicts -f -
-endif
+	$(KUSTOMIZE) build config/deployments/control-plane$(DEPLOY_TARGET) | kubectl apply --server-side --force-conflicts -f -
```

Comment:
These control-plane-a, control-plane-b, control-plane-c etc directories feel odd. Wouldn't it be easier if it was a tree struct? control-plane/a, /b, /c etc

Reply:
>Wouldn't it be easier if it was a tree struct?

How so? If we end up with a proliferation of control-plane-xxx dirs, then yeah from an organization standpoint it would be better. From a functional standpoint, I am not sure how there would be a difference.

Reply:
Discussed offline briefly with @burmanm and I agree that the tree structure is better. I'll update the PR accordingly.